### ISSUE TITLE
URL should contain the items when there is no items parameter

### ISSUE THREAD

**Issue by Kexort** [Kexort/relic-checklist#22]
When we visit https://5k-mirrors.github.io/relic-checklist without the items parameter the items in local storage should show up in the link.

@thisismydesign I think this would be the correct behavior, close the issue if it isn't

**Comment by thisismydesign** [comment]
Right, that makes sense. I wouldn't say this is critical though. Workaround: Modifying the selection fills the url params again.
